types/express-serve-static-core/express-serve-static-core-tests.ts

```diff
@@ -9,6 +9,8 @@ app.listen(3000, (err: any) => {
 app.get('/:foo', req => {
     req.params.foo; // $ExpectType string
     req.params[0]; // $ExpectType string
+    // $ExpectType string | false | null
+    req.is(['application/json', 'application/xml']);
```

Comment:
Can you please add some tests for the other possible types of the `type` argument?